docs/topics/Gateway.md

```diff
@@ -1446,6 +1449,20 @@ Sent when a [Slash Command](#DOCS_INTERACTIONS_SLASH_COMMANDS/) relevant to the
 
 Sent when a user in a guild uses a [Slash Command](#DOCS_INTERACTIONS_SLASH_COMMANDS/). Inner payload is an [Interaction](#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction).
 
+### Stage Instances
+
+#### Stage Instance Create
+
+Sent when a [Stage instance](#DOCS_RESOURCES_STAGE_INSTANCE) is created (i.e. the Stage is now "live"). Inner payload is a [Stage instance](#DOCS_RESOURCE_STAGE_INSTANCE/stage-instance-object)
+
+#### Stage Instance Update
+
+Sent when a [Stage instance](#DOCS_RESOURCES_STAGE_INSTANCE) has been updated. Inner payload is a [Stage instance](#DOCS_RESOURCE_STAGE_INSTANCE/stage-instance-object)
+
+#### Stage Instance Delete
+
+Sent when a [Stage instance](#DOCS_RESOURCES_STAGE_INSTANCE) has been deleted (i.e. the Stage has been closed). Inner payload is a [Stage instance](#DOCS_RESOURCE_STAGE_INSTANCE/stage-instance-object)
```

Comment:
Not sure why you're capitalizing "Stage". "Stage" isn't a proper noun, nor is it really referring to a product like the word "Discord" might (if that makes sense).

Reply:
Stage is actually used as a proper noun in our app surfaces, so I would like to keep it consistent. 
![image](https://user-images.githubusercontent.com/5456182/118347754-7d2f2f00-b4fa-11eb-9f02-7b62f4fca124.png)

Reply:
Well to be honest, it also seems odd to me that it's used as a proper noun in the app, but I think that reasoning is sound. Feel free to mark this comment as resolved.

(My other comment is about consistency in the table, so I don't think that one should be marked as resolved yet.)